bioio_base/reader.py

```diff
@@ -261,6 +282,39 @@ def set_scene(self, scene_id: Union[str, int]) -> None:
                 f"or integer (for scene index). Provided: {scene_id} ({type(scene_id)}."
             )
 
+    def set_resolution_level(self, resolution_level: int) -> None:
+        """
+        Set the resolution level.
+
+        Parameters
+        ----------
+        resolution_level: int
+            The resolution level to access the image at.
+
+        Raises
+        ------
+        TypeError
+            The provided value wasn't an integer.
+        """
+        # Ensure typing
+        if not isinstance(resolution_level, int):
```

Comment:
if `resolution _level` is already typed as an int, In what situation would this error be reached?

Reply:
Yeah idk I was kinda unsure about this too, largely just did it because `set_scene` does something similar, but taking another look it does feel unnecessary compared to why `set_scene` does it. I'll go ahead and remove it.